src/app/routes/utils/fetchPageData/utils/getRouteProps/index.js

```diff
@@ -22,7 +22,8 @@ const getNonCanonicalPlatformId = params => {
 };
 
 const getRouteProps = url => {
-  const matchedRoutes = matchRoutes(routes, url);
+  const urlWithoutQuery = url.split('?')?.[0];
```

Comment:
Should we consider using [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) instead of this? Something along the lines of:

```suggestion
  const { pathname: urlWithoutQuery } = new URL(`https://www.bbc.com${url}`);
```